src/plopfile_init.js

```diff
@@ -126,12 +126,23 @@ module.exports = function (plop, cfg) {
                     defaults.plugins &&
                     defaults.plugins.maven &&
                     defaults.plugins.maven.ignoreNamespacePrefix || '') + '\\.', 'g'), '')+'.v1'
-        }, {
+        },{
             type: 'input',
             name: 'mavenGroupId',
             message: 'Which Maven groupId do you want to use?',
             when: answers => answers.plugins.includes('maven'),
             default: (answers) => answers.javaPackage.split('.').slice(0,-2).join('.')
+        },{
+            type: 'list',
+            name: 'snapshotVersion',
+            message: 'Do you want to create Maven SNAPSHOT Versions?',
```

Comment:
I am not sure about having that in the list of questions. It is really technical and not something that usual API designer are able to answer.

Reply:
I have no opinion on that. Please tell me how you want it and I will change it accordingly.